src/components/hooks/useIsMobile.ts

```diff
@@ -0,0 +1,30 @@
+import { useState, useEffect } from "react";
+
+const useIsMobile = (
+  // Defaults at md: 768px
+  breakpoint = 768,
```

Comment:
Same here, there's a hooks for this with React-Use:
https://github.com/streamich/react-use/blob/master/docs/createBreakpoint.md

Reply:
This hook seem unnecessary and can be done like this:

```tsx
const isMobile = useBreakpoint() === "S"
```